NuGet.config

```diff
@@ -1,8 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <packageSources>
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-</configuration>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />

```

Comment:
@pranavkm if I'm not mistaken, the `dev` branch currently relies on dotnet-core 2.0. Only this package repo has the corresponding packages, or at least that's what I found.

The code didn't build for me last night without this. Snarky @friism was cursing under his breath and suspecting that people at Microsoft hadn't tested the build on a clean machine (or in a clean Docker container) and that the build worked on their machines because they have this (or an equivalent repo) in their global nuget.config.

I'm happy to be wrong.!